nautobot/core/tests/nautobot_config.py

```diff
@@ -49,3 +59,5 @@
 
 # Metrics need to enabled in this config as overriding them with override_settings will not actually enable them
 METRICS_ENABLED = True
+
+TEST_OUTPUT_DIR = os.getenv("NAUTOBOT_TEST_OUTPUT_DIR")
```

Comment:
Should this have a fallback if that variable is undefined?

Reply:
If not specified, the repo root directory is used for XML files. Do you have some proposal?

Reply:
Yup, it's working with both options.